### PR TITLE
Styling Setup and Tests

### PR DESCRIPTION
**Pull request by poema-cavalcanti**

### CODE REVIEW

File: test/client/App.test.js [hunk 1]
```diff
@@ -4,9 +4,10 @@ import renderer from 'react-test-renderer';
 import App from '../../client/components/App';
 
 describe('App component', () => {
-  it('renders', () => {
+  it('renders all three sections', () => {
```
Comment (janicelam21) [inline test/client/App.test.js]: what does 'all three sections' mean? all three sections of the reservations module?
@poema-cavalcanti

**Comment by janicelam21** [comment]
Just have that one question on your App.test.js file. Those tests look good!

**Comment by poema-cavalcanti** [comment]
> Just have that one question on your App.test.js file. Those tests look good!

Which question?

**Comment by janicelam21** [comment]
just tagged you in it

**Review by janicelam21 — APPROVED** [review]
Looks good!

**Pull request merged**